migrations/20200927132044-update-patient-barcode-non-nullable.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      `UPDATE patients SET "barcodeValue" = "id" WHERE "barcodeValue" IS NULL`
+    );
+    return queryInterface.changeColumn('patients', 'barcodeValue', {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true,
+    });
+  },
+  down: async (queryInterface, Sequelize) => {
+    await Promise.all([
```

Comment:
since it shouldn't matter the order that the constraint is removed and the column changed, so we can send run them consecutively to gain a (tiny) performance boost